client/signup/processing-screen/index.jsx

```diff
@@ -63,9 +64,14 @@ export class SignupProcessingScreen extends Component {
 	}
 
 	componentDidMount() {
+		const { flowName, localeSlug } = this.props;
+		const locale = localeSlug.split( /[-_]/ )[ 0 ];
```

Comment:
I am not sure if `localeSlug` could ever be `null` or `undefined`. Would it be better to add a condition here?

Reply:
I added a condition but honestly, don't believe it's possible in any way to reach this stage without a locale. Some similar functions [have that check.](https://github.com/Automattic/wp-calypso/blob/master/client/components/language-picker/utils.js#L82)